provider/cluster_rosa_classic_resource.go

```diff
@@ -686,13 +821,31 @@ func populateRosaClassicClusterState(ctx context.Context, object *cmv1.Cluster,
 	state.ConsoleURL = types.String{
 		Value: object.Console().URL(),
 	}
-	state.ComputeNodes = types.Int64{
+	state.Replicas = types.Int64{
 		Value: int64(object.Nodes().Compute()),
 	}
 	state.ComputeMachineType = types.String{
 		Value: object.Nodes().ComputeMachineType().ID(),
 	}
 
+	disableUserWorkload, ok := object.GetDisableUserWorkloadMonitoring()
+	if ok && disableUserWorkload {
+		state.DisableWorkloadMonitoring = types.Bool{
+			Value: true,
+		}
+	}
+	getAdditionalTrustBundle, ok := object.GetAdditionalTrustBundle()
+	if ok {
+		// TODO - how to retrieve the file name??
+		logger.Debug(ctx, getAdditionalTrustBundle)
```

Comment:
I think this is only the contents of the file as a string

Reply:
I removed that code - @oriAdler and I will decide tomorrow how to deal with additional trust bundle